inspirehep/modules/theme/views.py

```diff
@@ -101,3 +117,32 @@ def data():
 @blueprint.route('/ping')
 def ping():
     return 'OK'
+
+
+@blueprint.route('/postfeedback', methods=['POST', ])
+def postfeedback():
+    """Handler to create a ticket from user feedback."""
+    feedback = request.form.get('feedback')
+    if feedback == '':
+        return make_response(jsonify(success=False), 400)
+
+    replytoaddr = request.form.get('replytoaddr', None)
+    if replytoaddr is None:
+        if current_user.is_anonymous:
+            return make_response(jsonify(success=False), 403)
+        else:
+            replytoaddr = current_user.get('email')
+            if replytoaddr == '':
+                return make_response(jsonify(success=False), 403)
+
+    content = '''Feedback:
+{feedback}'''.format(feedback=feedback)
+    message = Message(sender=current_app.config['CFG_SITE_SUPPORT_EMAIL'],
+        recipients=[current_app.config['INSPIRELABS_FEEDBACK_EMAIL']],
+        subject='INSPIRE Labs Feedback',
+        body=content,
+        reply_to=replytoaddr)
+
+    send_email(message)
```

Comment:
You're right. I was seeing it on the console, but it wasn't doing the right thing: now that I added `delay` I get a serialization error:
```python
15:09:54 web.1       | Traceback (most recent call last):
15:09:54 web.1       |   File "/Users/jacquerie/.virtualenvs/invenio3/lib/python2.7/site-packages/flask/app.py", line 1836, in __call__
15:09:54 web.1       |     return self.wsgi_app(environ, start_response)
15:09:54 web.1       |   File "/Users/jacquerie/.virtualenvs/invenio3/lib/python2.7/site-packages/werkzeug/wsgi.py", line 659, in __call__
15:09:54 web.1       |     return app(environ, start_response)
15:09:54 web.1       |   File "/Users/jacquerie/.virtualenvs/invenio3/lib/python2.7/site-packages/flask/app.py", line 1820, in wsgi_app
15:09:54 web.1       |     response = self.make_response(self.handle_exception(e))
15:09:54 web.1       |   File "/Users/jacquerie/.virtualenvs/invenio3/lib/python2.7/site-packages/flask/app.py", line 1403, in handle_exception
15:09:54 web.1       |     reraise(exc_type, exc_value, tb)
15:09:54 web.1       |   File "/Users/jacquerie/.virtualenvs/invenio3/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
15:09:54 web.1       |     response = self.full_dispatch_request()
15:09:54 web.1       |   File "/Users/jacquerie/.virtualenvs/invenio3/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
15:09:54 web.1       |     rv = self.handle_user_exception(e)
15:09:54 web.1       |   File "/Users/jacquerie/.virtualenvs/invenio3/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
15:09:54 web.1       |     reraise(exc_type, exc_value, tb)
15:09:54 web.1       |   File "/Users/jacquerie/.virtualenvs/invenio3/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
15:09:54 web.1       |     rv = self.dispatch_request()
15:09:54 web.1       |   File "/Users/jacquerie/.virtualenvs/invenio3/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
15:09:54 web.1       |     return self.view_functions[rule.endpoint](**req.view_args)
15:09:54 web.1       |   File "/Users/jacquerie/.virtualenvs/invenio3/src/inspire-next/inspirehep/modules/theme/views.py", line 146, in postfeedback
15:09:54 web.1       |     send_email.delay(message)
15:09:54 web.1       |   File "/Users/jacquerie/.virtualenvs/invenio3/lib/python2.7/site-packages/celery/app/task.py", line 453, in delay
15:09:54 web.1       |     return self.apply_async(args, kwargs)
15:09:54 web.1       |   File "/Users/jacquerie/.virtualenvs/invenio3/lib/python2.7/site-packages/celery/app/task.py", line 565, in apply_async
15:09:54 web.1       |     **dict(self._get_exec_options(), **options)
15:09:54 web.1       |   File "/Users/jacquerie/.virtualenvs/invenio3/lib/python2.7/site-packages/celery/app/base.py", line 354, in send_task
15:09:54 web.1       |     reply_to=reply_to or self.oid, **options
15:09:54 web.1       |   File "/Users/jacquerie/.virtualenvs/invenio3/lib/python2.7/site-packages/celery/app/amqp.py", line 305, in publish_task
15:09:54 web.1       |     **kwargs
15:09:54 web.1       |   File "/Users/jacquerie/.virtualenvs/invenio3/lib/python2.7/site-packages/kombu/messaging.py", line 165, in publish
15:09:54 web.1       |     compression, headers)
15:09:54 web.1       |   File "/Users/jacquerie/.virtualenvs/invenio3/lib/python2.7/site-packages/kombu/messaging.py", line 241, in _prepare
15:09:54 web.1       |     body) = dumps(body, serializer=serializer)
15:09:54 web.1       |   File "/Users/jacquerie/.virtualenvs/invenio3/lib/python2.7/site-packages/kombu/serialization.py", line 164, in dumps
15:09:54 web.1       |     payload = encoder(data)
15:09:54 web.1       |   File "/usr/local/Cellar/python/2.7.11/Frameworks/Python.framework/Versions/2.7/lib/python2.7/contextlib.py", line 35, in __exit__
15:09:54 web.1       |     self.gen.throw(type, value, traceback)
15:09:54 web.1       |   File "/Users/jacquerie/.virtualenvs/invenio3/lib/python2.7/site-packages/kombu/serialization.py", line 59, in _reraise_errors
15:09:54 web.1       |     reraise(wrapper, wrapper(exc), sys.exc_info()[2])
15:09:54 web.1       |   File "/Users/jacquerie/.virtualenvs/invenio3/lib/python2.7/site-packages/kombu/serialization.py", line 55, in _reraise_errors
15:09:54 web.1       |     yield
15:09:54 web.1       |   File "/Users/jacquerie/.virtualenvs/invenio3/lib/python2.7/site-packages/kombu/serialization.py", line 164, in dumps
15:09:54 web.1       |     payload = encoder(data)
15:09:54 web.1       |   File "/Users/jacquerie/.virtualenvs/invenio3/lib/python2.7/site-packages/msgpack/__init__.py", line 47, in packb
15:09:54 web.1       |     return Packer(**kwargs).pack(o)
15:09:54 web.1       |   File "msgpack/_packer.pyx", line 231, in msgpack._packer.Packer.pack (msgpack/_packer.cpp:231)
15:09:54 web.1       |
15:09:54 web.1       |   File "msgpack/_packer.pyx", line 233, in msgpack._packer.Packer.pack (msgpack/_packer.cpp:233)
15:09:54 web.1       |
15:09:54 web.1       |   File "msgpack/_packer.pyx", line 192, in msgpack._packer.Packer._pack (msgpack/_packer.cpp:192)
15:09:54 web.1       |
15:09:54 web.1       |   File "msgpack/_packer.pyx", line 221, in msgpack._packer.Packer._pack (msgpack/_packer.cpp:221)
15:09:54 web.1       |
15:09:54 web.1       |   File "msgpack/_packer.pyx", line 228, in msgpack._packer.Packer._pack (msgpack/_packer.cpp:228)
15:09:54 web.1       |
15:09:54 web.1       | EncodeError: can't serialize <flask_mail.Message object at 0x101659cd0>
```